packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/handle-authentication-responses/react-native/handle-responses.md

```diff
@@ -0,0 +1,77 @@
+### Response handling
+
+The SDK provides two options, Promise and Event listener, to handle responses. You can choose either approach based on your preference.
+
+#### Handle response in Promise
+
+The `signIn` function will return a Promise if user credentials (username, password) have been provided. Success and error responses can be easily handled in standard Promise way.
```

Comment:
bolded my edits for these two sentences:

The `signIn` function **returns** a Promise if user credentials (username, password) **are** provided. Success and error responses can be easily handled in **a** standard Promise way.

Reply:
will change to `the standard Promise way` instead of `a standard Promise way`